core/controllers/pages_test.py

```diff
@@ -24,4 +24,3 @@ def test_about_page(self):
         response = self.testapp.get('/about')
         self.assertEqual(response.status_int, 200)
         self.assertEqual(response.content_type, 'text/html')
-        response.mustcontain('Credits', 'License')
```

Comment:
Since translations is a frontend item, we might want to tweak the tests here a little. Would it work if you check that the i18n tag is being passed to the front?